pkg/virt-controller/watch/vm.go

```diff
@@ -2820,8 +2819,6 @@ func (c *VMController) isTrimFirstChangeRequestNeeded(vm *virtv1.VirtualMachine,
 		// If we do not update `vmi` by asking the API Server this function could
 		// erroneously trim the just added StartRequest because it would see a running
 		// vmi with no DeletionTimestamp
-		vmi, _ := c.clientset.VirtualMachineInstance(vm.ObjectMeta.Namespace).Get(context.Background(), vm.GetName(), metav1.GetOptions{})
-		// If the current VMI is running, then it has been started.
 		if vmi != nil && vmi.DeletionTimestamp == nil {
```

Comment:
I know it is not related to these changes but i wonder why do we check if  `vmi.DeletionTimestamp == nil`